staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -269,6 +269,17 @@ type VolumeStats struct {
 	// Reference to the PVC, if one exists
 	// +optional
 	PVCRef *PVCReference `json:"pvcRef,omitempty"`
+
+	// VolumeHealthStats contains data about volume health
+	// +optional
+	VolumeHealthStats `json:"volumeHealthStats,omitempty"`
```

Comment:
Can you change this to a pointer?
VolumeHealthStats *VolumeHealthStats `json:"volumeHealthStats,omitempty"`